bakerydemo/settings/base.py

```diff
@@ -182,6 +182,15 @@
 MEDIA_ROOT = os.path.join(PROJECT_DIR, "media")
 MEDIA_URL = "/media/"
 
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "staticfiles": {
+        "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
```

Comment:
I think we should use `ManifestStaticFilesStorage` as the default, to match Wagtail's default project template